toolboxes/feel/feelmodels/modelcore/tabulateinformation.hpp

```diff
@@ -0,0 +1,61 @@
+/* -*- mode: c++; coding: utf-8; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4; show-trailing-whitespace: t -*- vim:fenc=utf-8:ft=cpp:et:sw=4:ts=4:sts=4
+ */
+
+#pragma once
+
+#include <feel/feelcore/feel.hpp>
+#include <tabulate/table.hpp>
+#include <feel/feelcore/json.hpp>
+
+namespace Feel
+{
+
+class TabulateInformationProperties
```

Comment:
how about having this kind of feature in feelpp ?

Reply:
idem as before